ui/app/styles/core/element-styling.scss

```diff
@@ -170,3 +170,10 @@ form {
 label {
   cursor: pointer;
 }
+
+// HDS modifications
+.hds-modal {
```

Comment:
I wasn't sure where to put this - if there are more I may create a separate file, but it seemed okay here for now